data/data/coreos/rhcos.json

```diff
@@ -1,107 +1,106 @@
 {
   "stream": "rhcos-4.16",
   "metadata": {
-    "last-modified": "2024-10-10T20:07:44Z",
```

Comment:
need a similar scos.json file for centos bootimages.

Reply:
Note this file is not going to be committed. It's just to exercise CI.